templates/create.tmpl.php

```diff
@@ -108,10 +108,10 @@
 			sort($users, SORT_NATURAL | SORT_FLAG_CASE );
                         foreach ($users as $user) : ?>
                             <li class="cl_user_item cl_access_item" id="user_<?php p($user); ?>" >
-				<?php p($user); ?>
-				(
 				<?php p(OC_User::getDisplayName($user)); ?>
```

Comment:
`OC_User::getDisplayName($user)` is [deprecated](https://doc.owncloud.org/api/classes/OCP.User.html#method_getDisplayName). You should use `$userMgr->get($user)->getDisplayName()` instead.